ROADMAP.md

```diff
@@ -19,33 +19,39 @@ The following is a list of prioritized roadmaps divided by OS (Linux/Windows) an
 
 **AMD Instinct**
```

Comment:
We could also directly link to https://github.com/ROCm/ROCm/discussions/4276 in the "prioritized target architectures" section right above this line.